src/index.js

```diff
@@ -1 +1,21 @@
-'use strict';
+import 'dotenv/config';
+import express from 'express';
+import cors from 'cors';
+
+import { roomRouter } from './routes/roomRouter.js';
+import { getChat } from './services/getChat.js';
+
+const PORT = process.env.PORT || 3005;
+
+const app = express();
+
+app.use(cors());
+
+app.use('/rooms', express.json(), roomRouter);
```

Comment:
You can divide this middleware for this controller to add it for every request.

```suggestion
app.use(express.json());

app.use('/rooms', roomRouter);
```

Reply:
As I mentioned in roomController comment the app.use can be simplified to 
```suggestion
app.use(roomRouter);
```